doc/source/try_examples.json

```diff
@@ -0,0 +1,3 @@
+{
+  "global_min_height": "600px"
+}
```

Comment:
We've documented this option here: https://jupyterlite-sphinx.readthedocs.io/en/latest/directives/try_examples.html#global-min-height